.github/workflows/default.yml

```diff
@@ -463,10 +463,18 @@ jobs:
     - name: Extract edge release version
       id: extract_edge_release_version
       run: |
-        release_version=$(git branch -r | grep -E 'origin/[0-9]{2}\.[0-9]{2}$' | awk -F'/' '{print $2}' | sort -V | tail -n 1)
-        echo "RELEASE_VERSION=$release_version" >> $GITHUB_OUTPUT
+        edge_release=$(git ls-remote --heads | grep -E 'refs/heads/[0-9]{2}\.[0-9]{2}$' | awk -F'/' '{print $3}' | sort -V | tail -n 1)
+        echo "EDGE_RELEASE=$edge_release" >> $GITHUB_OUTPUT
+        git ls-remote --heads | grep -q "refs/heads/${{ steps.extract_minor_version.outputs.RELEASE_VERSION }}\$"
+        if [ $? -eq 0 ]; then
+          is_undistributed=false
```

Comment:
"Undistributed" looks a strange konglish.
Please remember the principle: _Use negations sparingly._
Don't make an extra step of indirection and reasoning in the code reader's head.

In this case, you may just inverse the branches to rename it "is_distributed".

Reply:
Also, I'd like to be sure that the string value "false" is really evaluated as false in the followed conditional expression.